### DISCUSSION TITLE
Possible to create a select with options when creating resource?

### DISCUSSION THREAD

**Discussion by planetaska** [motor-admin/motor-admin#37] [General]
I find there is a Select type for columns, but it seems the Select type requires a Query to generate the options.
Is there a way to create old fashioned pre-defined string key-value paired options for the Select?

**Comment by omohokcoj** [comment]
@planetaska use 'Tag' type instead: https://demo.getmotoradmin.com/data/customers/5
<img width="1304" alt="image" src="https://user-images.githubusercontent.com/5418788/154459268-ee19702a-379b-496a-a5ba-e52b1570d5bb.png">

**Reply by planetaska** [reply]
That does it. Thank you!
I wonder if this means the key and value will have to be the same. But I guess I can figure out some workaround if that's not possible.
Great work. Loving the experience so far!

**Reply by jakubjanousek** [reply]
For us, it would also be great to have the option to state key/value pairs here. Some of the column values are not human-readable

**Reply by omohokcoj** [reply]
@jakubjanousek @planetaska it has been released in the latest version - you can specify key-value pairs for tag selector using csv fromat:
```
tech,Technology
bio,Biolab
```

**Reply by jakubjanousek** [reply]
@omohokcoj this is amazing. Thank you very much!
Just one small thing, would it be possible to also allow 0 as an value?
Unfortunately, we have that as a valid db value in such enums and it doesn't display anything in MotorAdmin with the latest change.

**Reply by omohokcoj** [reply]
@jakubjanousek good catch! it has been fixed in the 0.2.76 😄
